Updated Rhelp Plugin

martinbndr opened the pull request:
- Subcommands of Command Groups also get shown in the embed like on the ?help command (actually forgot to add that)
- Check if a found command is not a Cog (cog help not supported)
- Using the command replies to the user with a message and then sends the embed
- Some fixes Hope this is working

martinbndr committed:
Updated Rhelp Plugin

- Subcommands of Command Groups also get shown in the embed like on the ?help command (actually forgot to add that)
- Check if a found command is not a Cog (cog help not supported)
- Using the command replies to the user with a message and then sends the embed
- Some fixes
Hope this is working

StephenDaDev commented:
The changes need to be to the rhelp.py file in the rhelp folder, not a new file in root of repo

StephenDaDev reviewed (changes requested):
The changes need to be to the rhelp.py file in the rhelp folder, not a new file in root of repo

martinbndr committed:
Delete rhelp.py

martinbndr committed:
Upload rhelp to the correct dir #1

- Subcommands of Command Groups also get shown in the embed like on the ?help command (actually forgot to add that)
- Check if a found command is not a Cog (cog help not supported)
- Using the command replies to the user with a message and then sends the embed
- Some fixes Hope this is working

martinbndr commented:
Fixed

StephenDaDev committed:
Update rhelp.py

StephenDaDev reviewed (approved):
Added translations of strings in German

Added general help message & credited author

Approved w/ changes
